.github/workflows/ci.yml

```diff
@@ -24,28 +25,28 @@ on:
       - '.github/workflows/ci.yml'
 
 jobs:
-  build:
+  run_syntax_tests:
     name: Run syntax_tests
-    runs-on: ubuntu-18.04
+    strategy:
+      max-parallel: 2
+      matrix:
+        latest-sublime-build: [4113, 4116] # stable & dev channel
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
-
-      - name: Download syntax_tests
-        run: wget -O st_syntax_tests.tar.xz https://download.sublimetext.com/st_syntax_tests_build_4116_x64.tar.xz
-
-      - name: Extract syntax_tests
-        run: tar xf st_syntax_tests.tar.xz
-
-      - name: Move syntax_tests
-        run: mv st_syntax_tests/* ./
+      - name: Git checkout
+        uses: actions/checkout@master
 
-      - name: Cleanup syntax_tests archive
-        run: rm -R st_syntax_tests st_syntax_tests.tar.xz
+      - name: Get syntax_tests binary for Sublime Text Build ${{ matrix.latest-sublime-build }}
+        run: |
+          wget -O st_syntax_tests.tar.xz https://download.sublimetext.com/st_syntax_tests_build_${{ matrix.latest-sublime-build }}_x64.tar.xz
```

Comment:
I seem to recall that there used to be an endpoint which always returned the latest version - I wonder if it is still around and if we want to use it. It would save us having to update it here after every release...

Reply:
I guess you meant these?

- <https://www.sublimetext.com/updates/4/stable_update_check>
- <https://www.sublimetext.com/updates/4/dev_update_check>

Reply:
One one hand, we could have a bot (like dependabot ?) that opens a PR if the dev and stable version increments. But on the other hand, once the CI workflow is well defined, it's just the matter of updating version numbers. I wonder if automation is worth it for this case.